src/main/java/mate/academy/bookstore/BookstoreApplication.java

```diff
@@ -5,9 +5,7 @@
 
 @SpringBootApplication
 public class BookstoreApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(BookstoreApplication.class, args);
-	}
-
+    public static void main(String[] args) {
+        SpringApplication.run(BookstoreApplication.class, args);
+    }
 }
```

Comment:
![image](https://github.com/dmytro-voronin/jv-book-store/assets/63095184/38514f66-6bb9-4f4b-a6f0-ace92f86d012)
Fix this plz